src/test.rs

```diff
@@ -103,13 +103,13 @@ fn test_heap_alloc_and_free() {
 
 #[test]
 fn test_empty_frame_allocator() {
-    let mut frame = FrameAllocator::new();
```

Comment:
should default to 32 ?

Reply:
It looks like the default only applies if the type is specified, for some reason.